README.md

```diff
@@ -3,36 +3,25 @@
 
 # Virtual Machine Packages
 
-## Overview
-This repository contains the source code for packages supporting the projects below:
-* [FLARE-VM](https://github.com/mandiant/flare-vm)
-* [Commando-VM](https://github.com/mandiant/commando-vm)
-* [ThreatPursuit-VM](https://github.com/mandiant/ThreatPursuit-VM)
-
-> NOTE: Packages do not contain actual software distributions. Packages are PowerShell scripts that only contain instructions for obtaining distributions. See [Chocolatey legal information](https://docs.chocolatey.org/en-us/information/legal) for more details.
-
-### Package Lifecycle
```

Comment:
shortened here and copied this to https://github.com/mandiant/VM-Packages/wiki/Package-Lifecycle as well